CMakeLists.txt

```diff
@@ -178,9 +178,10 @@ add_subdirectory(examples/random)
 add_subdirectory(examples/histograming)
 add_subdirectory(examples/async)
 add_subdirectory(examples/misc)
+add_subdirectory(examples/phys)
 
 if(Minuit2_FOUND)
-add_subdirectory(examples/phys)
```

Comment:
All examples in the  directory '''phys''' depends on Minuit2. 
You need to put it inside the guards ''' if(Minuit2_FOUND) ... endif(Minuit2_FOUND)'''